### PR TITLE
Show that atom keys enable dot access

### PR DESCRIPTION
**Pull request by nathanl**

### CODE REVIEW

File: about_atoms.exs [hunk 1]
```diff
@@ -15,6 +15,15 @@ defmodule AboutAtoms do
     assert __? == list[:name]
   end
 
+  think "Atom keys enable a different access syntax" do
```
Comment (iamvery) [inline about_atoms.exs]: Might be worth just stating it as is

> Atom keys may be accessed with dot syntax

**Comment by iamvery** [comment]
I love it! Thank you ❤️

File: about_atoms.exs [hunk 2]
```diff
@@ -15,6 +15,15 @@ defmodule AboutAtoms do
...
+    map = %{name: "Jay"}
+    assert map.name == __?
+
+    map = %{"name" => "Jay"}
+    attempt = fn -> __? end
+    assert_raise KeyError, "key :name not found in: %{\"name\" => \"Jay\"}", attempt
```
Comment (iamvery) [inline about_atoms.exs:24]: Also we might illustrate that dot syntax fails for a missing key, where Access (subscript) syntax is more forgiving by returning `nil`.

Comment (nathanl) [inline about_atoms.exs:24]: Good idea

File: about_atoms.exs (hunk 1 above)
Comment (nathanl) [inline about_atoms.exs]: Sounds good

File: about_atoms.exs (hunk 2 above)
Comment (nathanl) [inline about_atoms.exs:24]: Added a test for that

**Comment by iamvery** [comment]
Thank you so much for these! Love 'em ❤️ 

I made a couple changes when I merged it, FYI 7bae2f303083382f372dce51f4cb6b437e639002 5b6a1db289a7d1feaa0ed2a7d8e23762f148aef3

**Comment by nathanl** [comment]
👍

**Pull request merged**